src/components/AnchorForCommentsOnly/anchorForCommentsOnlyPropTypes.js

```diff
@@ -21,8 +21,8 @@ const propTypes = {
     /** Any children to display */
     children: PropTypes.node,
 
-    /** Display label in case of attachments */
-    fileName: PropTypes.string,
+    /** Display label in case of attachments. Text to display in case of anchors */
```

Comment:
Can you improve this comment? 
Post some suggestions in the comment before commiting any changes!

Reply:
What you think of:
`/** Display label in case of attachments or text in case of anchors */`

Reply:
/** Filename in case of attachments, anchor text in case of URLs or emails. */

cc: @marcochavezf What do you say?

Reply:
LGTM the latter `/** Filename in case of attachments, anchor text in case of URLs or emails. */` it's more descriptive

Reply:
Commited!